src/app/public/src/modules/sidebar/sidebar-wrapper.component.html

```diff
@@ -0,0 +1,14 @@
+<div class="stache-sidebar-wrapper" [ngClass]="{'sidebar-closed': sidebarClosed, 'sidebar-open': !sidebarClosed  }">
+  <div class="stache-sidebar-header">
+    <button *ngIf="!sidebarClosed" class="sky-btn sky-btn-link-inline stache-sidebar-button" (click)="closeSidebar()" aria-label="sidebar close">
+      <i class="fa fa-chevron-left" aria-hidden="Close"></i>
+    </button>
+    <button *ngIf="sidebarClosed" class="sky-btn sky-btn-link-inline stache-sidebar-button" (click)="openSidebar()" aria-label="sidebar open">
+      <i class="fa fa-chevron-right" aria-hidden="Open"></i>
+    </button>
+  </div>
+  <div class="stache-sidebar-content">
```

Comment:
Is the sidebar hidden from screen readers when collapsed? If so, you'll need to add the `aria-expanded` roles, etc. for "accordion" content.

See: https://www.w3.org/TR/wai-aria-practices-1.1/#accordion